Session-4/GenericStack/Stack.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace GenericStack
+{
+  internal class Stack<T>
+  {
+    private List<T> _data = (List<T>) null;
+
+    public Stack() => this._data = new List<T>();
+
+    public void PushElement(T item) => this._data.Add(item);
+
+    public void PopElement() => this._data.RemoveAt(this._data.Count<T>() - 1);
+
+    public void DisplayElements()
```

Comment:
- Rename the method as GetData.
- It should return string
- Why are you reversing the array? Instead you can iterate from N-1 to 0 right.
- Remove console.writeline and return string